hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -310,37 +298,37 @@ checkClose ctx parties initialUtxoHash sig cperiod headPolicyId =
           , headId
           , contesters
           } -> (snapshotNumber, utxoHash, p, contestationDeadline, headId, contesters)
-      _ -> traceError "H07"
+      _ -> traceError $(errorCode WrongStateInOutputDatum)
 
   checkSnapshot
     | closedSnapshotNumber > 0 =
-      traceIfFalse "H11" $
-        verifySnapshotSignature parties closedSnapshotNumber closedUtxoHash sig
+        traceIfFalse $(errorCode InvalidSnapshotSignature) $
+          verifySnapshotSignature parties closedSnapshotNumber closedUtxoHash sig
     | otherwise =
-      traceIfFalse "H12" $
-        closedUtxoHash == initialUtxoHash
+        traceIfFalse $(errorCode ClosedWithNonInitialHash) $
+          closedUtxoHash == initialUtxoHash
```

Comment:
I would revert to the original code formatting so that we avoid future commit that will just reformat the old way because people are using the nix version of it.